src/components/IdCard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function IdCard(props){
+    const {lastName, firstName, gender, height, birth, picture} = props;
+    return(
+        
+        <div className="idCard">
+            <img src={picture} alt="profile pic" />
+            <div className="id">
+            <p className="firstname">First Name: {firstName}</p>
+            <p className="lasttname">Last Name: {lastName}</p>
+            <p className="gender">Gender: {gender}</p>
+            <p className="height">Height: {height}</p>
+            <p className="birth">Birth: {birth.toLocaleString()}</p>
```

Comment:
Here, to not show the hours, you could've used slice, like:

`<p className="birth">Birth: {birth.toLocaleString().slice(0,10)}</p>`